clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -533,6 +533,19 @@
             "None"
           ],
           "description": "Formatting settings"
+        },
+        "cobol-lsp.lspconfig.compiler": {
+          "type": "string",
+          "default": "",
```

Comment:
How about specifying the default "IGYCRCTL"?

Reply:
That would enable the override by default. Suppressing any future changes we might do without first clearing the field.